src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -806,6 +806,7 @@ def _prepare_4d_causal_attention_mask_with_cache_position(
         device: torch.device,
         cache_position: torch.Tensor,
         batch_size: int,
+        **kwargs,
```

Comment:
Some models have extra kwargs. With `**kwargs` we can make the generalization :)